tasks/mod_security.yml

```diff
@@ -1,12 +1,16 @@
 ---
-- yum: name={{ item }} state=installed
+- package:
+    name: "{{ item }}"
+    state: present
   with_items:
   - mod_security
   - mod_security_crs
   notify: verify config and restart httpd
   name: Install mod_security packages
 
-- lineinfile: dest=/etc/httpd/conf.d/mod_security.conf regexp='^(\s*)SecRuleEngine ' line='\1SecRuleEngine {{ 'DetectionOnly' if mod_sec_detection_only else 'On' }}'
+- lineinfile:
+    dest: /etc/httpd/conf.d/mod_security.conf
+    regexp: "^(\\s*)SecRuleEngine "
+    line: "\\1SecRuleEngine {{ 'DetectionOnly' if mod_sec_detection_only else 'On' }}"
   notify: verify config and restart httpd
-  when: mod_sec_detection_only is True
```

Comment:
true, missed reading the description. nevertheless it should be in a separate PR, or in a patchset PR so you can review each part individually